sorc/build_ufs.sh

```diff
@@ -44,6 +44,14 @@ COMPILE_NR=0
 CLEAN_BEFORE=YES
 CLEAN_AFTER=NO
 
+#TODO temp patch for build update for noaacload in advance of updating ufs_module.fd repo for global-workflow building
+if [[ "${MACHINE_ID}" == "noaacloud" ]] ; then
+    patched=$(grep upp-addon-env modulefiles/ufs_noaacloud.intel.lua; echo $?)
+    if [[ ${patched} == "1" ]] ; then
+       patch -R modulefiles/ufs_noaacloud.intel.lua ../ufs_noaacloud.intel.diff
+    fi   
+fi
+
```

Comment:
Same question as one I also left in the newly added diff file. Can the need for this patch be resolved before merging this PR?

Reply:
Work is needed in the ufs-weather-model to eliminate the application of the diff.  This is is a work around.